### PR TITLE
Added temperature control

### PR DESCRIPTION
**Pull request by turangarusso**
Now you can also control the temperature parameter via the library

### CODE REVIEW

**Review by rchatham — APPROVED** [review]
Looks good!

File: Sources/OpenAISwift/OpenAISwift.swift [hunk 1]
```diff
@@ -24,9 +24,9 @@ extension OpenAISwift {
     ///   - model: The AI Model to Use. Set to `OpenAIModelType.gpt3(.davinci)` by default which is the most capable model
     ///   - maxTokens: The limit character for the returned response, defaults to 16 as per the API
     ///   - completionHandler: Returns an OpenAI Data Model
-    public func sendCompletion(with prompt: String, model: OpenAIModelType = .gpt3(.davinci), maxTokens: Int = 16, completionHandler: @escaping (Result<OpenAI, OpenAIError>) -> Void) {
+    public func sendCompletion(with prompt: String, model: OpenAIModelType = .gpt3(.davinci), maxTokens: Int = 16, temperature: Double, completionHandler: @escaping (Result<OpenAI, OpenAIError>) -> Void) {
```
Comment (adamrushy) [inline Sources/OpenAISwift/OpenAISwift.swift:27]: It's worth adding the default value or make it optional to respect the API default

**Comment by adamrushy** [comment]
@turangarusso please check my comments :] thanks for the contribution 🎉

File: Sources/OpenAISwift/OpenAISwift.swift (hunk 1 above)
Comment (turangarusso) [reply Sources/OpenAISwift/OpenAISwift.swift:27]: Yeah right

**Comment by turangarusso** [comment]
> @turangarusso please check my comments :] thanks for the contribution 🎉

Now the temperature parameter has assigned the default value :D, thank you !

**Pull request merged**